octoprint_mrbeam/templates/conversion_dialog.jinja2

```diff
@@ -663,9 +663,9 @@
                             <i class="icon-file-text" aria-hidden="true"></i> {{ _(' Learn how to do this.') }}</a>
                     </p>
                     <img id="laserhead_focus_reminder_image" src="/plugin/mrbeam/static/img/focus_reminder.png">
-                    <div id="laserhead_focus_reminder_alert"><i class="fa fa-exclamation-triangle" aria-hidden="true"
-                                                                style="color: #e25303;"></i>
-                        {{ _('Remember to turn off your Mr Beam if you disconnect the ribbon cable!') }}
+                    <div id="laserhead_focus_reminder_alert">
+                        <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+                        <span>{{ _('Reminder: Never remove any components while your Mr Beam is still active! To make sure you’ve turned your Mr Beam off beforehand, remove the key and unplug it from the power source.') }}</span>
```

Comment:
Please 

1. use `you have` instead of the informal `you've`
2. remove -> removed
3. unplug -> unplugged